srcs/httpMethod/HTTPMethod.cpp

```diff
@@ -292,37 +288,8 @@ int do_CGI(std::string &response_message_str,
     response_message_str = CreateErrorResponse(500, err_pages); 
     return (500);
   }
-  int cgi_out_stream = cgi.GetCgiSocket();
-
-  /*
-   * Read CGI output

```

Comment:
read cgi output has been moved to here https://github.com/oooooorriiiii/webserv/blob/c7f0a3e2401879bf35da1f229d0cc43fc3ca2402/srcs/server/socket.cpp#L287
Which is called from here after the fd is checked by poll https://github.com/oooooorriiiii/webserv/blob/c7f0a3e2401879bf35da1f229d0cc43fc3ca2402/srcs/server/socket.cpp#L157